src/firejail/profile.c

```diff
@@ -1507,7 +1507,7 @@ int profile_check_line(char *ptr, int lineno, const char *fname) {
 		else if (strncmp(ptr, "rlimit-as ", 10) == 0) {
 			cfg.rlimit_as = parse_arg_size(ptr + 10);
 			if ( cfg.rlimit_as == 0 ){
-				perror("Error: invalid rlimit-as size in profile file. use only non-negative numbers and k,m,g suffix for size");
+				perror("Error: invalid rlimit-as in profile file. only use positive numbers and k, m or g suffix.");
```

Comment:
It's a capital O (not o) after completing a sentence:
perror("Error: invalid rlimit-as in profile file. Only use positive numbers and k, m or g suffix.");
Please change this and we're set for merge.